tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateTestHelpers.hpp

```diff
@@ -0,0 +1,10 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
```

Comment:
Is this file necessary? Why not include the 5 headers directly in the test?

Reply:
No, it doesn't serve much of a purpose. I only added it so that the number of `#include`s in `Test_TensorExpressions.cpp`. In my next PR, there will be a similar suite of tests for a new function, and that will be another 5 includes on top of these 5 for testing `evaluate`. So, that was really the only reason!

Let me know if you would like me to remove this file and just include the 5 helper files directly in `Test_TensorExpressions.cpp`. I don't feel strongly about it either way

Reply:
If this were 100 includes, I would definitely say to keep the extra file, but for 5 includes (and even with 5 more in the PR), personally I think I'd prefer just putting the include lines directly in the main test? It simplifies the process of looking up where functions are defined if there are fewer includes of includes...

Reply:
Will do! Less work on the reader for tracing back through documentation

Reply:
Done!